plugin/cmd/kube-scheduler/app/server.go

```diff
@@ -110,6 +122,11 @@ func (s *SchedulerServer) AddFlags(fs *pflag.FlagSet) {
 	fs.Float32Var(&s.KubeAPIQPS, "kube-api-qps", s.KubeAPIQPS, "QPS to use while talking with kubernetes apiserver")
 	fs.IntVar(&s.KubeAPIBurst, "kube-api-burst", s.KubeAPIBurst, "Burst to use while talking with kubernetes apiserver")
 	fs.StringVar(&s.SchedulerName, "scheduler-name", s.SchedulerName, "Name of the scheduler, used to select which pods will be processed by this scheduler, based on pod's annotation with key 'scheduler.alpha.kubernetes.io/name'")
+
+	fs.BoolVar(&s.LeaderElect, "leader-elect", s.LeaderElect, "Start a leader election client and gain leadership before executing scheduler loop.")
```

Comment:
A driveby thought which may or may not be a good idea: If this is going to be a common pattern, consider putting the code for setting up these flags into the leaderelection package?